examples/plot_benchmark_grid_search.py

```diff
@@ -0,0 +1,67 @@
+"""
+=======================
+Benchmarking with MOABB with Grid Search
+=======================
```

Comment:
```suggestion
========================================
Benchmarking with MOABB with Grid Search
========================================
```

Dashes and equal signs for title and header should match the length of title in ReST format, otherwise it complains when building doc.